app/dcs/api/uplink-cli/share-command/page.md

```diff
@@ -169,15 +169,15 @@ You can also generate a URL to share your projects/buckets/objects
 {% code-group %}
 
 ```windows
-./uplink.exe share sj://cakes/ --url --not-after=none --base-url=https://link.storjshare.io
+./uplink.exe share sj://cakes/ --url --not-after=none
```

Comment:
Wondering if we should add `--not-after=none` to the custom domain/tls docs... it'll error if this flag isn't set, right?

Reply:
yeah, I think it will. I'll add that in